source/features/remove-projects-tab.tsx

```diff
@@ -68,8 +68,9 @@ void features.add(__filebasename, {
 		pageDetect.isOrganizationProfile
 	],
 	exclude: [
-		// Repo/Organization owners should see the tab. If they don't need it, they should disable Projects altogether
+		// Repo/Organization owners/Owm profile should see the tab. If they don't need it, they should disable Projects altogether
 		pageDetect.canUserEditRepo,
+		pageDetect.isOwnUserProfile,
 		pageDetect.canUserEditOrganization
```

Comment:
Typos, meaningless comment edit, lack of test URL, no "mention of the issue in a JS comment" as suggested… 

```suggestion
		// Repo/Organization owners should see the tab. If they don't need it, they should disable Projects altogether
		pageDetect.canUserEditRepo,
		pageDetect.canUserEditOrganization,

		// Also keep it visible on your own profile due to #3737
		pageDetect.isOwnUserProfile
```